torch/nn/modules/conv.py

```diff
@@ -119,7 +118,7 @@ class Conv1d(_ConvNd):
          In other words, for an input of size :math:`(N, C_{in}, L_{in})`, if you want a
          depthwise convolution with a depthwise multiplier `K`,
          then you use the constructor arguments
-         :math:`(\text{in_channels}=C_{in}, \text{out_channels}=C_{in} * K, ..., \text{groups}=C_{in})`
+         :math:`(in\_channels=C_{in}, out\_channels=C_{in} * K, ..., groups=C_{in})`
```

Comment:
please put back the `\text{...}` wrapper

Reply:
I can put these back...I'm on the fence about the \text{...} wrappers though. Variable names should be in italics, and adding \text{...} just makes the code less readable. That said, its a matter of preference.

Reply:
Also finding there are situations where \text{...} has to be escaped; like when used after \prod

ie 

\prod(\\text{kernel\_size})

note the two slashes 

It's not clear to me that using \text{...} is "worth it".

Reply:
I'd vote for keeping the `\text`. My rule of thumb is that if the variable has more than one letter, then it should ideally not be on italic

Reply:
I agree with @fmassa . Please keep the `\text`. The purpose here is to make it read nice. Readability is prioritized over code tidiness here.

Also from playing on https://khan.github.io/KaTeX/, I don't seem to need to escape `\` after `\prod(`

Reply:
@goodlux I think this still needs to be resolved